web/packages/teleterm/src/ui/App.test.tsx

```diff
@@ -99,25 +94,15 @@ test('activating a workspace via deep link overrides the previously active works
     })
   );
 
-  render(
-    <MockAppContextProvider appContext={appContext}>
-      <ConnectionsContextProvider>
-        <VnetContextProvider>
-          <ResourcesContextProvider>
-            <AppInitializer />
-          </ResourcesContextProvider>
-        </VnetContextProvider>
-      </ConnectionsContextProvider>
-    </MockAppContextProvider>
-  );
+  render(<App ctx={appContext} />);
```

Comment:
Now that handling of deep links is done outside of `AppContext` and `AppInitializer`, we have to render the whole app instead.